CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "CYGWIN")
   add_definitions(-DSPIRV_WINDOWS)
 elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
   add_definitions(-DSPIRV_MAC)
+elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "iOS")
+  add_definitions(-DSPIRV_MAC)
```

Comment:
It is probably better to define `SPIRV_IOS` macro to be consistent with other platforms and in case you will need to implement something iOS-specific in the future.
There is [only single place in the code](https://github.com/KhronosGroup/SPIRV-Tools/search?q=SPIRV_MAC&unscoped_q=SPIRV_MAC) where  `SPIRV_MAC` macro is used, and where `SPIRV_IOS` will need to be added:

```cpp
#if defined(SPIRV_ANDROID) || defined(SPIRV_LINUX) || defined(SPIRV_MAC) || \
    defined(SPIRV_IOS) || defined(SPIRV_FREEBSD) || defined(SPIRV_EMSCRIPTEN)
```